G29.2 Graph Visualization

alexandrevenancio21 opened the pull request:
2 classes added to package BuildPropertyGraph

alexandrevenancio21 committed:
build property graph using JungGraph, code with javadocs.
2 classes added to package BuildPropertyGraph

Ptome2000 commented on src/main/java/BuildPropertyGraph/PropertyGraphBuilder.java:
Class still necessary? Seeing as it is no longer being used

alexandrevenancio21 committed:
build property graph using JungGraph, code with javadocs.
2 classes added to package BuildPropertyGraph. class PropertyGraphBuilder.java deleted
